public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+  
+  ### Establishing an input property binding
+  
+  To create an input property binding, we need to carry out these steps:
+  1. Adding an input property to the child component (apply the `@Input()` decorator on the particular property)
+  1. Using this property with the _input binding syntax_ in the parent template to bind it to an expression
+  
+  The Hero Job Board application uses input data binding to pass the information about an invited hero from `HeroJobBoard` (parent) to all
+  `HeroPanel` children. The `HeroJobBoard` component is the one that knows the circle of heroes who should be invited to listen to job requests, and it embeds 
+  a `HeroPanel` component instance for each hero.
+  
+  In this scenario, `HeroPanel` declares an input property, `hero`, as the `@Input()` decorator suggests:
+  
++makeExample('component-communication/ts/app/invite-heroes/hero-panel.ts', null, 'invite-heroes/hero-panel.ts', {pnk: /(@Input\(\))/g})
+
+:marked
+  `HeroJobBoard` can iterate through its `invitedHeroes` collection and set up its `HeroPanel` children so that a single hero is
+  assigned to each of them through the `hero` input property. As we learned before, the Angular property binding syntax 
+  (property name wrapped into square brackets) is used:
+
++makeExample('component-communication/ts/app/invite-heroes/hero-job-board.ts', null, 'invite-heroes/hero-job-board.ts', {pnk: /(\[hero\])/g})
+  
+:marked
+  The `Hero.heroes` static collection contains six heroes for demonstration purposes. When the application starts, `invitedHeroes` is empty, 
+  but clicking the __Invite heroes__ button invokes `inviteHeroes()` and fills up the `invitedHeroes` array. According the template definition, 
+  these bindings are established between `HeroJobBoard` and its `HeroPanel` children:
+  
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/parent-and-children.png" alt="Input data binding")
+  
+:marked
+  When we run the app and click the **Invite heroes** button, the application UI reflects this stucture:
+
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/heroes-invited-ui.png" alt="Heroes are invited")
+  
+:marked
+  ### Using `@Input`
+  
+  `Input` is defined in the core module of Angular, and we need to import it so that we can use it:
+  
++makeExample('component-communication/ts/app/invite-heroes/hero-panel.ts', 'import', null, {blk: /(Input)/g})
+
+:marked
+  `@Input()` always must be used with parentheses:
+
++makeExample('component-communication/ts/app/invite-heroes/hero-panel.ts', 'input-def', null, {pnk: /(@Input\(\))/g})
+
+:marked
+.alert.is-critical
+  :marked
+    It's easy to forget the parentheses. Our application will fail mysteriously if we do.
+:marked
+  Instead of using the `@Input()` annotation, we can use the `inputs` annotation property of `@Component()` to specify input properties.
```

Comment:
It is not an "annotation property".  It's a property of the `@Component `configuration object.